Documentation/security.md

```diff
@@ -143,6 +143,14 @@ $ etcd -name infra2 -data-dir infra2 \
 
 The etcd members will form a cluster and all communication between members in the cluster will be encrypted and authenticated using the client certificates. You will see in the output of etcd that the addresses it connects to use HTTPS.
 
+## Notes For etcd Proxy
+
+etcd proxy terminates the TLS from its client if the connection is secure, and uses proxy's own key/cert specified in `--peer-key-file` and `--peer-cert-file` to communicate with etcd members.
+
+Proxy communicates with etcd member through both advertise client urls and advertise peer urls of that member. It forwards the client requests to etcd members’ advertise client urls, and it syncs the initial cluster configuration through etcd members’ advertise peer urls.
+
+So when the client authentication is enabled for etcd member, user must ensure that the peer cert specified (`--peer-cert-file`) for the proxy is valid for client authentication. Also when the peer authentication is enabled, user must ensure that the peer cert specified for proxy is valid for peer authentication.
```

Comment:
"When client authentication is enabled for an etcd member, the administrator must ensure that the peer certificate specified in the proxy's `--peer-cert-file` option is valid for that authentication. The proxy's peer certificate must also be valid for peer authentication if peer authentication is enabled."